src/main/java/com/couchbase/lite/replicator/Replication.java

```diff
@@ -93,7 +94,42 @@
     /**
      * Properties of the replicator that are saved across restarts
      */
-    protected Map<ReplicationField, Object> properties;
+    private Map<ReplicationField, Object> properties;
+
+    /**
+     * Currently only used for test
+     */
+    public Map<String, Object> getProperties() {
```

Comment:
As this only for testing, I think you can use package-private (no explicit modifier) for this method.